src/components/lead/CdrLead.vue

```diff
@@ -25,7 +25,6 @@ const baseClass = 'cdr-lead'
 
 <template>
   <div :class="style[baseClass]">
-    <div :class="style[`${baseClass}__top`]" />
     <div
       v-if="imgSrc"
```

Comment:
still question if this should be picture - I guess as a srcset can just be used in image this is fine?